otherlibs/unix/unix.mli

```diff
@@ -243,10 +243,15 @@ val execvpe : string -> string array -> string array -> 'a
 
 val fork : unit -> int
 (** Fork a new process. The returned integer is 0 for the child
-   process, the pid of the child process for the parent process.
+   process, the pid of the child process for the parent process. It
+   fails if the OCaml process is multi-core (any domain has been
+   spawned). In addition, if any threads have been spawned then the
+   child process might be in a corrupt state.
```

Comment:
This isn't entirely clear on how other threads have been spawned, given the invariant earlier. Do you mean system threads independently spawned of the OCaml runtime?

Reply:
Yes, I think this sentence refers to pthreads that would have been created by C code or other foreign code in the current process.

Reply:
Also systhreads, which is what I meant originally. C/FFI programmers already know not to mix fork & threads. I clarified the comment.